js/sidePanel.js

```diff
@@ -0,0 +1,94 @@
+const maxHpElement = document.querySelector('#MaxHp');
+const levelElement = document.querySelector('#Level');
+const xpElement = document.querySelector('#Xp');
+const coinsElement = document.querySelector('#Coins');
+const strengthElement = document.querySelector('#Strength');
+const enduranceElement = document.querySelector('#Endurance');
+const pointsElement = document.querySelector('#Points');
+
+const strengthButton = document.querySelector('#StrengthButton');
+const enduranceButton = document.querySelector('#EnduranceButton');
+const confirmButton = document.querySelector('#ConfirmButton');
+
+class Player {
+    constructor(name, hp, strength, endurance) {
+        this.name = name;
+        this.hp = hp;
+        this.strength = strength;
+        this.endurance = endurance;
+
+        this.xp = 0;
+        this.maxXp = 6;
+        this.coins = 0;
+        this.level = 0;
+        this.points = 5;
+
+        this.strengthPoint = 0;
+        this.endurancePoint = 0;
+    }
+}
+
+const players = {
```

Comment:
let's name this variable as playerTypes.
Create a new variable:
```suggestion
const player = playerTypes.strongman;
```
And use this variable instead of `players.strongman.strength`